pymc/tests/test_shape_handling.py

```diff
@@ -438,13 +438,12 @@ def test_mvnormal_shape_size_difference(self):
             rv = pm.MvNormal.dist(mu=[1, 2, 3], cov=np.eye(3), size=(5, 4))
             assert tuple(rv.shape.eval()) == (5, 4, 3)
 
-        # When using `size` the API behaves like Aesara/NumPy
-        with pytest.warns(
-            ShapeWarning,
-            match=r"You may have expected a \(2\+1\)-dimensional RV, but the resulting RV will be 5-dimensional",
-        ):
+        # The support dimensions are excluded from size. This is a difference w.r.t. NumPy
+        # and from 2021-05 through 2022-02 the behavior here matched NumPy but warned about
+        # potentially unexpected results.
```

Comment:
I find this comment confusing. The support dimensions being excluded from size is the official Numpy API, see size explanation for multivariate normal: https://numpy.org/doc/stable/reference/random/generated/numpy.random.multivariate_normal.html

I would also remove the from x to y behavior...

Reply:
I left it as an explanation for why there's a `pytest.warns(None)` below. But I can take it out